docs/xdr/features/collect/integrations/endpoint/paloalto_cortex_edr.md

```diff
@@ -0,0 +1,66 @@
+uuid: b5968b0a-1206-4d3b-b960-e6b61e7f6ee6
+name: Palo Alto Cortex EDR
+type: intake
+
+
+## Overview
+
+Palo Alto Cortex EDR is an advanced Endpoint Detection and Response solution offering real-time threat detection, investigation, and response capabilities, empowering organizations to proactively defend against sophisticated cyber threats across their endpoints.
+
+This setup guide shows how to forward EDR events collected on the Palo Alto Cortex XDR platform to Sekoia.io.
+
+{!_shared_content/operations_center/detection/generated/suggested_rules_9b95c9cf-8b78-4830-a1ed-b9e88f05e67a_do_not_edit_manually.md!}
+
+{!_shared_content/operations_center/integrations/generated/9b95c9cf-8b78-4830-a1ed-b9e88f05e67a.md!}
+
+
+
+## Create API Key
+
+Before using the **Cortex XDR** connector, you must generate an API key in Cortex XDR. For that you need to follow steps bellow :
+
+1. Within **Cortex XDR**, click **Settings**.
+2. Click **Configurations**.
+
+    ![Configuration Page](/assets/operation_center/integration_catalog/cloud_and_saas/paloalto_cortex_edr/configuration.png){: style="max-width:100%; height: 500px;"}
+
+3. In the **Configurations** menu, select **API Keys**, which appears in the **Integrations** section.
+4. In the API Keys window, click **+New Key** button to create a new API key. And the **Generate API Key** window will open.
+
+    ![Api Key Page](/assets/operation_center/integration_catalog/cloud_and_saas/paloalto_cortex_edr/api_key.png){: style="max-width:100%"}
+
+5. Select the **Standard** Security Level.
+6. Select a role for the API key. After selecting the role, click **Generate**.
```

Comment:
Which role the customer should assign to the API key?

Reply:
Yes they put that we need `Endpoint Management` permission. But as I don't have access to see the api key part in my platform I don't know if it's exactly the same name.